webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/fields/DownloadUtility.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  FormGroup,
+  FormSelect,
+  FormSelectOption,
+} from '@patternfly/react-core';
+
+import { translate as __ } from '../../../../../common/I18n';
+
+export const DownloadUtilities = ['curl', 'wget'];
```

Comment:
Add `wget` to https://github.com/theforeman/foreman/blob/develop/.eslintrc#L17, so GA doesn't spam with warnings.